src/shared/ui/Avatar/utils/generateInitials.ts

```diff
@@ -0,0 +1,8 @@
+export function generateInitials(value = '') {
+  return value
+    .split(' ')
+    .slice(0, 2)
```

Comment:
Do we really need slice here? I assume initials should be generated from a string following "Name Surname" pattern (so we don't need `slice(0, 2)`), but I might be missing something

Reply:
The username can have many spaces (not just one, there are no restrictions), and we need a maximum of 2 letters.